benchmarks/views/user.py

```diff
@@ -396,10 +397,12 @@ def _is_instance_path(path: str, plugin: str) -> bool:
 
 def extract_identifiers(zip_ref):
     # define patterns for each plugin type (data and metrics to be added later)
-    possible_plugins = ["models", "benchmarks"]
+    possible_plugins = ["models", "benchmarks", "data", "metrics"]
     registry_patterns = {
         "models": re.compile(r"model_registry\['(.+?)'\]"),
         "benchmarks": re.compile(r"benchmark_registry\['(.+?)'\]"),
+        "data": re.compile(r"data_registry\['(.+?)'\]"),
+        "metrics": re.compile(r"metrics_registry\['(.+?)'\]"),
```

Comment:
```suggestion
        "metrics": re.compile(r"metric_registry\['(.+?)'\]"),
```
https://github.com/brain-score/vision/blob/master/brainscore_vision/__init__.py#L21